### PR TITLE
notices section

### PR DESCRIPTION
**Pull request by emnlpstr**
this PR contain parts that concern notices section

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [x] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

### CODE REVIEW

File: src/layouts/IndexPage.jsx [hunk 1]
```diff
@@ -49,9 +43,49 @@ export const IndexPage = () => {
   const classes = useStyles();
   const [, dispatch] = useContext(GlobalStateContext);
 
+  const [newsList, setNewsList] = useState([]);
+
   useEffect(() => {
-    announce('Pagina caricata ' + name);
+    (async () => {
+      try {
+        await fetch(`https://raw.githubusercontent.com/tuxolino/testapi/main/data/NC002.json`)
```
Comment (sebbalex) [inline src/layouts/IndexPage.jsx:51]: ?

File: src/layouts/IndexPage.jsx [hunk 2]
```diff
@@ -49,9 +43,49 @@ export const IndexPage = () => {
...
+          .then((data) => {
+            setNewsList(data.windows);
+          });
+      } catch (error) {
+        console.log(error);
+      }
+    })();
   }, []);
+
+  // API rest to get SF TOKEN
```
Comment (sebbalex) [inline src/layouts/IndexPage.jsx:62]: what is it for?

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: src/layouts/IndexPage.jsx (hunk 2 above)
Comment (emnlpstr) [reply src/layouts/IndexPage.jsx:62]: It's an error, I removed the comment block

File: src/layouts/IndexPage.jsx (hunk 1 above)
Comment (emnlpstr) [reply src/layouts/IndexPage.jsx:51]: It's for the notices section, we call the data exposed from salesforce team

Comment (emnlpstr) [reply src/layouts/IndexPage.jsx]: I updated the url

**Review by sebbalex — APPROVED** [review]

**Pull request merged**